src/coreComponents/physicsSolvers/PhysicsSolverBase.cpp

```diff
@@ -1209,6 +1209,13 @@ void PhysicsSolverBase::setupSystem( DomainPartition & domain,
   solution.create( dofManager.numLocalDofs(), MPI_COMM_GEOS );
 }
 
+void PhysicsSolverBase::setSparsityPattern( DomainPartition & GEOS_UNUSED_PARAM( domain ),
+                                            DofManager & dofManager,
+                                            SparsityPattern< globalIndex > & pattern )
+{
+  dofManager.setSparsityPattern( pattern );
```

Comment:
At some point we may want to adopt a consistent style and chose between `GEOS_UNUSED_PARAM` and `GEOS_UNUSED_VAR`.